docs/troubleshoot/app-framework/zss-error-codes.md

```diff
@@ -367,3 +367,15 @@ The following error message codes may appear on ZSS log. Use the following messa
 
 No action required.
 
+### ZWES0054E
+
+  ZIS AUX Server started in wrong key 8
+
+  **Reasons:**
+
+  authPluginLib was missing in the `ZWESASTC JCL` and the `ZWESISTS JCL`.
+
+  **Action:**
+ 
+  The `IZPSRGSP`,`IZPSRGAD`, and `ZWESVUSR` need read access to the profile `OMVSAPPL`. Then the ZWESASTC started task was started successfully by the ZWESISTS started task.
```

Comment:
IZP is some third party thing, not part of Zowe.
And `ZWESVUSR` is just the default name of Zowe's user account.

This should read as:

> When the Zowe STC userid has read access to the profile SAF `OMVSAPPL`, the ZWESASTC task will be able to successfully be started on an as-needed basis from ZWESISTC.